shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/Export.java

```diff
@@ -7,8 +7,8 @@
 import java.lang.annotation.Target;
 
 /**
- * Annotation to indicate that a method in {@link Variables} is exported to the
- * Shesmu compiler
+ * Annotation to indicate that a method in {@link GsiStdValue} is exported to
+ * the Shesmu compiler
```

Comment:
Only for `GsiStdValue`, or for all Input Formats?

Reply:
All of them. Fixed.